src/App.js

```diff
@@ -1,5 +1,5 @@
 import React from "react"
-import HorrorMovies from "./components/HorrorMovies"
+import HorrorMovies from "./components/HorrorMovies.js"
```

Comment:
Still have the js here. It doesn't break anything but it is inconsistent with the rest of your code.